src/main/java/frc/robot/controls/GameControllerDriverConfig.java

```diff
@@ -19,6 +22,7 @@
  */
 public class GameControllerDriverConfig extends BaseDriverConfig {  
   private final GameController kDriver = new GameController(Constants.DRIVER_JOY);
+  private final XboxController xbox = new XboxController(5);
```

Comment:
This should not be here. This class is for the driver's controller only; put the controller for subsystems in Operator.java, which doesn't exist yet.This should be in a separate PR.

Reply:
done